br/tests/rawkv/main.go

```diff
@@ -201,7 +206,11 @@ func (t *RawKVBRTester) InjectFailpoint(failpoint string) error {
 
 func (t *RawKVBRTester) ExecBRCmd(ctx context.Context, cmdStr string) ([]byte, error) {
 	log.Info("exec br cmd", zap.String("br", t.br), zap.String("args", cmdStr))
-	cmd := exec.CommandContext(ctx, t.br, strings.Split(cmdStr, " ")...)
+	cmdParameter := []string{fmt.Sprintf("-test.coverprofile=%s_%s_%d", *coverageFile, t.name, t.runCnt)}
```

Comment:
How about just create a unique temporary file (see [os.CreateTemp](https://pkg.go.dev/os#CreateTemp)) for the coverage ? Then we don't need the case name & run count.

(If use temporary file, it may need to clear the coverage output path before running test)

Reply:
It looks better. I have changed to temp file for coverage file. PTAL~